docs/reference/indices/aliases.asciidoc

```diff
@@ -107,7 +107,8 @@ In this case, the alias is a point-in-time alias that will group all
 current indices that match, it will not automatically update as new
 indices that match this pattern are added/removed.
 
-It is an error to index to an alias which points to more than one index.
+Keep in mind that indexing is a single index operation.  Therefore, you will receive an IllegalArgumentException
```

Comment:
We can certainly remove the actual exception reference in case it changes in the future.  The reason for this change is that we often get questions on why indexing requests fail when issued against an alias with multiple indices.  So this attempts to clarify the reason in documentation upfront.  If we think the existing description is sufficient, we can close this PR :)